examples/cfd/09_Darcy_flow_equation.ipynb

```diff
@@ -0,0 +1,537 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        a.data[:] = perm[:]
</pre><p><code>initialize_function</code> is recommended here for <code>a</code> at least so that the FD halo is padded with the edge values because currently it assumes that <code>a</code> is zero (not physical and potentially numerically dangerous).</p><p> </p><p><code>initialize_function(a, perm, 0)</code> should do it</p><p> </p><p>typo: pseudo not psuedo</p><p> </p><p>"the same data"? Do you mean initialize <code>a, f</code> with inputs permeability and forcing? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1998/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>